irohad/torii/processor/impl/stub_query_processor.cpp

```diff
@@ -25,7 +25,9 @@ namespace iroha {
     using dao::QueryResponse;
     using dao::Client;
 
-    QueryProcessorStub::QueryProcessorStub() {
+    QueryProcessorStub::QueryProcessorStub(ametsuchi::WsvQuery &wsv,
+                                           ametsuchi::BlockQuery &block) :
```

Comment:
Is it possible to separate the instance of Ametsuchi to WsvQuery and BlockQuery?
In other words, is `wsv_(wsv), block_(block)` better than `wsv_(ametsuchi), block_(ametsuchi)`?

And also, the interface of TxStore is now in BlockQuery because of downloading txs from blocks in flat files, is it right?